src/router/router.tsx

```diff
@@ -14,6 +17,14 @@ export default function Router() {
     {
       path: PROJECTS_PATH_EXAMPLE,
       element: <Projects />
+    },
+    // {
+    //   path: LOGIN_PATH,
+    //   element: <Login />
+    // },
+    {
+      path: MAIN_PATH,
+      element: <Main />
```

Comment:
here for `MAIN_PATH` do something like that to test your layout.

```
path: MAIN_PATH,
element: <DefaultLayout><h1>Hello world</h1></DefaultLayout />
```

Reply:
Fixed